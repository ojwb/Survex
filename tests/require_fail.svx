; pos=fail
; require a version later than the current one
*require 1.5
*fix a reference 0 0 0
