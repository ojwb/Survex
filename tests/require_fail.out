./require_fail.svx:2:13: error: End of line not blank
 *require 1.0abc ; test warning with suffix (added in 1.4.11)
             ^~~
./require_fail.svx:3:13: error: End of line not blank
 *require 1. 42 ; spaces in the version rejected in 1.4.11 and later
             ^~
./require_fail.svx:5:10: error: Survex version 1.5 or greater required to process this survey data.
 *require 1.5  ; require a version later than the current one
          ^~~
